src/predefined/continuous_famous_systems.jl

```diff
@@ -1405,3 +1498,297 @@ end
     du2 = 1/τ*(-N + G)
     return SVector{2}(du1, du2)
 end
+
+"""
+```julia
+sakarya(u0= [-2.8976045, 3.8877978, 3.07465];
+    a =-1.0,
+    b = 1.0,
+    c = 1.0,
+    h = 1.0,
+    p = 1.0,
+    q = 0.4,
+    r = 0.3,
+    s = 1.0
+)
+```
+```math
+\\begin{aligned}
+\\dot{x} &= ax + hy + syz\\\\
+\\dot{y} &= -by - px + qxz \\\\
+\\dot{z} &= cz - rxy
+\\end{aligned}
+```
+A system presenting robust chaos that varies from single wing to double wings to four wings.
+Its attractor arises due to merging of two disjoint bistable attractors [^Li2015].
+
+[^Li2015]:
+    Li, Chunbiao, et al (2015). A novel four-wing strange attractor born in bistability.
+    IEICE Electronics Express 12.4.
+"""
+function sakarya(u0= [-2.8976045, 3.8877978, 3.07465];
+    a =-1.0,
+    b = 1.0,
+    c = 1.0,
+    h = 1.0,
+    p = 1.0,
+    q = 0.4,
+    r = 0.3,
+    s = 1.0
+)
+    return CDS(loop_sakarya, u0, [a,b,c,h,p,q,r,s])
+end
+
+function loop_sakarya(u, p, t)
+    @inbounds begin
+        a, b, c, h, p, q, r, s = p
+        du1 = a*u[1] + h*u[2] + s*u[2]*u[3]
+        du2 = -b*u[2] - p*u[1] + q*u[1]*u[3]
+        du3 = c*u[3] - r*u[1]*u[2]
+    end
+    return SVector{3}(du1, du2, du3)
+end
+
+
+"""
+```julia
+lorenz_bounded(u0=[-13.284881, -12.444334, 34.188198];
+    beta = 2.667,
+    r = 64.0,
+    rho = 28.0,
+    sigma = 10.0
+)
+```
+```math
+\\begin{aligned}
+\\dot{X} &= \\sigma(Y-X)f(X,Y,Z) \\\\
+\\dot{Y} &= (-XZ + \\rho X -Y)f(X,Y,Z) \\\\
+\\dot{Z} &= (XY - \\beta Z)f(X,Y,Z)
+\\end{aligned}
+```
+
+Lorenz system bounded by a confining potential [^SprottXiong2015].
+
+[^SprottXiong2015]: Sprott & Xiong (2015). Chaos.
+"""
+function lorenz_bounded(u0=[-13.284881, -12.444334, 34.188198];
+    beta = 2.667,
+    r = 64.0,
+    rho = 28.0,
+    sigma = 10.0
+)
+    return CDS(loop_lorenzbounded, u0, [beta,r,rho,sigma])
+end
+
+function loop_lorenzbounded(u, p, t)
+    @inbounds begin
+        beta, r, rho, sigma = p
+        f = 1 - (u[1]^2 + u[2]^2 + u[3]^2) / r^2
+        du1 = sigma * (u[2] - u[1])*f
+        du2 = (u[1]*(rho - u[3]) - u[2])*f
+        du3 = (u[1]*u[2] - beta*u[3])*f
+    end
+    return SVector{3}(du1, du2, du3)
+end
+
+"""
+```
+lorenz_coupled(u0=[7.0921483,10.117408,25.708428,2.1689893,0.96121877,0.89550187];
+    beta = 2.667,
+    eps = 2.85,
+    rho = 28,
+    rho1 = 35,
+    rho2 = 1.15,
+    sigma = 10
+)
+```
+```math
+\\begin{aligned}
+\\dot{x}_1 &= \\sigma (y_1 - x_1)\\\\
+\\dot{y}_1 &= x_1 (\\rho _1 - z_1) - y_1 \\\\
+\\dot{z}_1 &= x_1 y_1 - \\beta z_1 \\\\
+\\dot{x}_2 &= \\sigma (y_2 - x_2) + \\eps (x_1 - x_2) \\\\
+\\dot{y}_2 &= x_2 (\\rho _2 - z_2) - y_2
+\\dot{z}_2 &= x_2 y_2 - \\beta z_2
+\\end{aligned}
+```
+Two coupled Lorenz attractors [^Lorenz1963].
+
+[^Lorenz1963]:
+    Lorenz, Edward N. Deterministic nonperiodic flow.
+    Journal of the atmospheric sciences 20.2 (1963): 130-141.
+"""
+function lorenz_coupled(u0=[7.0921483,10.117408,25.708428,2.1689893,0.96121877,0.89550187];
```

Comment:
You can safely remove this system. In DynamicalSystems.jl 3.0 I will create a "CoupledSystem" dynamical system that takes in two systems and couples them either uni or bi-directionally.